### DISCUSSION TITLE
How to access (via Domain) a Directus installation on a DigitalOcean droplet?

### DISCUSSION THREAD

**Discussion by rowild** [directus/directus#17812] [Archive]
Hi! I am new to DigitalOcean an also rather new to Docker and Docker Compose. 

I would like to setup Directus on DigitalOcean, and for the lack of better knowledge I chose a Droplet for doing so (not really knowing how a droplet differs from an app etc on DigitalOcean, even though I read through a lot of documents).

I followed this tutorial to setup my Droplet:
https://didcoding.com/tutorial/setting-up-a-droplet-and-installing-docker/

Then I cd'ed into my sudo-user's home folder at `/home/rowild/`, where I created a `www` folder which should hold my projects, the first one being `jvds-be`. So the `$(pwd)`is `/home/rowild/www/jvds-be/`.

In the PWD I created the `docker-compose.yml` that the documentation recommends:
https://docs.directus.io/self-hosted/quickstart.html
and run `docker-compose up -d`, which seems to do its job, everything is installed.

But know I have no idea how to access this installation from my browser. What do I have to do now?

**Comment by luciandavidescu** [comment]
Point your domain's DNS to the droplet's IP – either from your registrar/hosting or via a 3rd party like Cloudflare

**Comment by rowild** [comment]
@luciandavidescu Thank you for your response! Will look into that!
I read something about "ondigitalocean.app" domains that should be assigned automatically to one's app:
https://docs.digitalocean.com/products/app-platform/how-to/manage-domains/
but there is nothing in my backend that reveals such a domain. Am i missing something?

Also, why can the IP of a droplet not be used, at least for testing?
